e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts

```diff
@@ -55,77 +83,49 @@ class RegistrationDetails {
     phoneNumber: string,
     fsp: string,
   ) {
-    await expect(
-      this.page.locator(this.personalInformationTable),
-    ).toBeVisible();
-    expect(
-      await this.page.locator(this.personalInformationTable).isVisible(),
-    ).toBe(true);
-    expect(
-      await this.page.locator(this.personAffectedName).textContent(),
-    ).toContain(name);
-    expect(
-      await this.page.locator(this.personAffectedStatus).textContent(),
-    ).toContain(status);
-    expect(
-      await this.page
-        .locator(this.personAffectedStatusDateUpdate)
-        .textContent(),
-    ).toContain(date);
-    expect(
-      await this.page.locator(this.primaryLanguage).textContent(),
-    ).toContain(language);
-    expect(await this.page.locator(this.phoneNumber).textContent()).toContain(
-      phoneNumber,
+    await expect(this.personalInformationTable).toBeVisible();
+    expect(await this.personalInformationTable.isVisible()).toBe(true);
+    expect(await this.personAffectedName.textContent()).toContain(name);
+    expect(await this.personAffectedStatus.textContent()).toContain(status);
+    expect(await this.personAffectedStatusDateUpdate.textContent()).toContain(
+      date,
     );
-    expect(
-      await this.page.locator(this.financialServiceProvider).textContent(),
-    ).toContain(fsp);
+    expect(await this.primaryLanguage.textContent()).toContain(language);
+    expect(await this.phoneNumber.textContent()).toContain(phoneNumber);
+    expect(await this.financialServiceProvider.textContent()).toContain(fsp);
   }
 
   async openEditPaPopUp() {
-    await this.page.click(this.showAllButton);
+    const showAllButton = this.page.getByTestId('show-all-button');
```

Comment:
Is there a reason this isn't defined on the page? (ie. `this.showAllButton` like all the other locators)

Reply:
Sorry skipped my attention previously when I was testing it. Fixed!